app/navigation/ExpensesNavigator.scala

```diff
@@ -41,94 +42,108 @@ import javax.inject.{Inject, Singleton}
 @Singleton
 class ExpensesNavigator @Inject() () {
 
-  private val normalRoutes: Page => UserAnswers => TaxYear => BusinessId => Call = {
+  private val normalRoutes: Page => UserAnswers => TaxYear => BusinessId => Option[AccountingType] => Call = {
 
     case OfficeSuppliesAmountPage =>
       userAnswers =>
         taxYear =>
           businessId =>
-            userAnswers.get(OfficeSuppliesPage, Some(businessId.value)) match {
-              case Some(OfficeSupplies.YesAllowable) =>
-                officeSupplies.routes.OfficeSuppliesCYAController.onPageLoad(taxYear, businessId.value)
-              case Some(OfficeSupplies.YesDisallowable) =>
-                officeSupplies.routes.OfficeSuppliesDisallowableAmountController.onPageLoad(taxYear, businessId.value, NormalMode)
-              case _ =>
-                standard.routes.JourneyRecoveryController.onPageLoad()
-            }
+            _ =>
+              userAnswers.get(OfficeSuppliesPage, Some(businessId.value)) match {
+                case Some(OfficeSupplies.YesAllowable) =>
+                  officeSupplies.routes.OfficeSuppliesCYAController.onPageLoad(taxYear, businessId.value)
+                case Some(OfficeSupplies.YesDisallowable) =>
+                  officeSupplies.routes.OfficeSuppliesDisallowableAmountController.onPageLoad(taxYear, businessId.value, NormalMode)
+                case _ =>
```

Comment:
it generally correct to use _, but if you have few cases it is safer to name them
```
None | Some(OfficeSupplies.No) =>
```

what you get is that in future when you have to add one more option, the place you forgot to handle will not compile and you will have to make concious decision what you should do for the new type.